secomplican/secomplican/settings.py

```diff
@@ -132,16 +131,36 @@
 
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/dev/howto/static-files/
+STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
 STATIC_URL = '/static/'
 
 STATICFILES_DIRS = (
     os.path.join(BASE_DIR, 'static'),
 )
 
+# Simplified static file serving.
+# https://warehouse.python.org/project/whitenoise/
+STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
+
 LOGIN_URL = 'login'
 LOGIN_REDIRECT_URL = '/'
 LOGOUT_REDIRECT_URL = '/'
 
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'console': {
+            'class': 'logging.StreamHandler',
+        },
+    },
+    'loggers': {
+        'django': {
+            'handlers': ['console'],
+            'level': os.getenv('DJANGO_LOG_LEVEL', 'ERROR'),
+        },
+    },
+}
 
 # Esto hace que rompan los tests de staticfiles por alguna razón.
-# django_heroku.settings(locals())
+django_heroku.settings(locals(), logging=False, databases=not DEBUG)
```

Comment:
`logging=False` porque quiero que me loguee las cosas el logger que setee, y `databases=not DEBUG` porque quiero que en `DEBUG` las databases las setee sin SSL. Si no pongo eso, me tira un error.